Unity3D/Assets/RosSharp/Scripts/VelocitySubscriber.cs

```diff
@@ -0,0 +1,45 @@
+﻿using UnityEngine;
+using System.Collections;
+
+namespace RosSharp.RosBridgeClient
+{
+
+	[RequireComponent(typeof(RosConnector))]
+	public class VelocitySubscriber : MonoBehaviour
+	{
+		public GameObject Model;
```

Comment:
Please rename the variables as follows:
* Model -> UrdfModel
* twist_msg -> geometryTwist
* updateCmdVel -> updateVelocity